packages/visual-editor/src/components/puck/GetDirections.tsx

```diff
@@ -92,17 +92,15 @@ const GetDirections = ({
       fieldId={coordinateField.field}
       constantValueEnabled={coordinateField.constantValueEnabled}
     >
-      <Button
-        asChild
-        variant={variant}
+      <CTA
+        label={"Get Directions"}
+        link={searchQuery ?? "#"}
+        linkType={"URL"}
```

Comment:
There is an underlying DRIVING_DIRECTIONS linkType for the Link component. You'll need to update CTA accordingly (there are other linkTypes as well).